packages/components/skeleton/src/skeleton.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect, fixture } from '@open-wc/testing';
+import { html } from 'lit';
+import '../register.js';
+import { Skeleton } from "./skeleton.js";
+
+describe('sl-skeleton', () => {
+  let el: Skeleton;
+
+  beforeEach(async () => {
+    el = await fixture(html`<sl-skeleton></sl-skeleton>`);
+  });
+
+  it('should render correctly', () => {
+    expect(el).shadowDom.to.equalSnapshot();
+  });
+
+  it('should have an aria busy', () => {
+    expect(el).to.have.attribute('aria-busy', 'true');
+  });
+
+  it('should have a shimmer effect by default', () => {
+    expect(el).to.have.attribute('effect', 'shimmer');
+  });
+
+  it('should have a shimmer effect when set', async () => {
+    el.setAttribute('effect', 'shimmer');
```

Comment:
You're testing if the attribute is set, *after you've set the attribute* :)

Better to do `el.effect = 'shimmer';` and then test the value of the attribute :)